src/views/Dashboard/components/LatestSupportTickets/LatestSupportTickets.js

```diff
@@ -0,0 +1,283 @@
+              const ticketTempObject = {
+                ...ticketObject,
+                ticket: {
+                  ...ticketObject.ticket,
+                  content: JSON.parse(content)
+                }
+              };
+              addTicketAction(ticketTempObject);
+            });
+          })
+          .catch(e => console.error(e.message));
+      } else {
+        throw new Error('Error retrieving tickets.');
+      }
+    }
+  };
+
+  const isAdmin = roles => roles.includes(ADMIN);
+
+  useEffect(() => {
+    // noinspection JSUnresolvedVariable
+    const { isAuthenticated } = userObject;
+    if (isAuthenticated) {
+      const roles = userObject.roles || [];
+      getTicketsList(isAdmin(roles)).catch(e => console.error(e.message));
+    }
+  }, [userObject]);
+
+  const handleSortTicketsByDate = () => {
+    const dateFormat = 'DD-MM-YYYY';
+    if (sortByDateDesc) {
+      setSearchResults([
+        ...searchResults.sort((t1, t2) => (
+          moment(t1.ticket.createDate, dateFormat) - moment(t2.ticket.createDate, dateFormat)
+        ))
+      ]);
+    } else {
+      setSearchResults([
+        ...searchResults.sort((t1, t2) => (
+          moment(t2.ticket.createDate, dateFormat) - moment(t1.ticket.createDate, dateFormat)
+        ))
+      ]);      
+    }
+
+    setSortByDateDesc(!sortByDateDesc);
+    setPage(0);
+  };
+
+  const handleSortTicketsByStatus = () => {
+    if (sortByOpenStatus) {
+      setSearchResults([...searchResults.sort((t1, _) => t1.ticket.closed ? -1 : 1)]);
+    } else {
+      setSearchResults([...searchResults.sort((t1, _) => t1.ticket.closed ? 1 : -1)]);
+    }
+
+    setSortByOpenStatus(!sortByOpenStatus);
+    setPage(0);
+  };
+
+  const handlePageChange = (_event, page) => {
+    setPage(page);
+  };
+
+  const handleRowsPerPageChange = event => {
+    setPage(0);
+    setRowsPerPage(event.target.value);
+  };
+
+  return (
+    <Card {...rest} className={clsx(classes.root, className)}>
+      <CardHeader title="Latest Support Tickets" />
+      <Divider />
+      <CardContent className={classes.content}>
+        <PerfectScrollbar>
+          <div className={classes.inner}>
+            {(!searchResults.length && !hasTickets) ?
+              <Spinner /> :
+              <Table>
+                <TableHead>
+                  <TableRow>
+                    <TableCell>ID</TableCell>
+                    <TableCell>Author</TableCell>
+                    <TableCell sortDirection="desc">
+                      <Tooltip enterDelay={300} title="Sort">
+                        <TableSortLabel
+                          active
+                          direction={sortByDateDesc ? 'desc' : 'asc'}
+                          onClick={handleSortTicketsByDate}>
+                            Date
+                        </TableSortLabel>
+                      </Tooltip>
+                    </TableCell>
+                    <TableCell sortDirection="desc">
```

Comment:
If you want, you can add the padding = {'checkbox'} parameter here, but if you like the current look more, ignore this comment 😁

Reply:
Ok, looks better, added :+1: